### PR TITLE
sql-statements: update create view and drop view

### PR DESCRIPTION
**Pull request by SunRunAway**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: sql-statements/sql-statement-create-view.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 title: CREATE VIEW
 summary: TiDB 数据库中 CREATE VIEW 的使用概况。
 category: reference
-aliases: ['/docs-cn/dev/reference/sql/statements/create-view/']
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-view.md:5]: 删掉这个 alias 后，文档重构前的这篇文档的链接 `https://pingcap.com/docs-cn/dev/reference/sql/statements/create-view/` 将变成死链。所以这个 alias 不能删除。

File: sql-statements/sql-statement-create-view.md [hunk 2]
```diff
@@ -152,10 +151,13 @@ ERROR 1105 (HY000): insert into view v1 is not supported now.
 
 ## MySQL 兼容性
 
-* 目前 TiDB 中的视图不可插入且不可更新。
+* 目前 TiDB 中的视图不可插入。 `WITH CHECK OPTION` 只做了语法兼容但不生效。
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-view.md:154]:
```suggestion
* 目前 TiDB 中的视图不可插入。`WITH CHECK OPTION` 只做了语法兼容但不生效。
```

File: sql-statements/sql-statement-create-view.md [hunk 3]
```diff
@@ -152,10 +151,13 @@ ERROR 1105 (HY000): insert into view v1 is not supported now.
...
+* 目前 TiDB 中的试图不可更新，即不支持 `ALTER VIEW`，但可以使用 `CREATE OR REPLACE` 替代。
```
Comment (TomShawn) [inline sql-statements/sql-statement-create-view.md:155]:
```suggestion
* 目前 TiDB 中的视图不可更新，即不支持 `ALTER VIEW`，但可以使用 `CREATE VIEW` 或者 `REPLACE VIEW` 替代。
```

File: sql-statements/sql-statement-drop-view.md [hunk 4]
```diff
@@ -2,7 +2,6 @@
 title: DROP VIEW
 summary: TiDB 数据库中 DROP VIEW 的使用概况。
 category: reference
-aliases: ['/docs-cn/dev/reference/sql/statements/drop-view/']
```
Comment (TomShawn) [inline sql-statements/sql-statement-drop-view.md:5]: ditto，不能删除 alias。

**Comment by TomShawn** [comment]
LGTM

File: sql-statements/sql-statement-create-view.md [hunk 5]
```diff
@@ -152,10 +152,13 @@ ERROR 1105 (HY000): insert into view v1 is not supported now.
 
 ## MySQL 兼容性
 
-* 目前 TiDB 中的视图不可插入且不可更新。
+* 目前 TiDB 中的视图不可插入。`WITH CHECK OPTION` 只做了语法兼容但不生效。
+* 目前 TiDB 中的视图不可更新，即不支持 `ALTER VIEW`，但可以使用 `CREATE OR REPLACE` 替代。
```
Comment (zz-jason) [inline sql-statements/sql-statement-create-view.md:156]: 原文的不可插入以及不可更新指的应该是 mysql 的 insertable 和 updatable view：https://dev.mysql.com/doc/refman/8.0/en/view-updatability.html

Comment (SunRunAway) [reply sql-statements/sql-statement-create-view.md]: Updated.

Comment (zz-jason) [reply sql-statements/sql-statement-create-view.md]: @SunRunAway 看看这个 comment 呢

**Pull request merged**